shimmingtoolbox/unwrap/prelude.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+"""
+Wrapper to FSL Prelude (https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FUGUE/Guide#PRELUDE_.28phase_unwrapping.29)
+"""
+
+import glob
+import os
+import nibabel as nib
+import pathlib
+import tempfile
+import logging
+
+from shimmingtoolbox.utils import run_subprocess
+
+
+def prelude(wrapped_phase, mag, affine, mask=None, threshold=None, is_unwrapping_in_2d=False):
+    """wrapper to FSL prelude
+
+    This function enables phase unwrapping by calling FSL prelude on the command line. A mask can be provided to mask
+    the phase image provided. 2D unwrapping can be turned off. The output path can be specified. The temporary niis
+    can optionally be saved.
+
+    Args:
+        wrapped_phase (numpy.ndarray): 2D or 3D radian numpy array to perform phase unwrapping. (2 pi interval)
+        mag (numpy.ndarray): 2D or 3D magnitude numpy array corresponding to the phase array
+        affine (numpy.ndarray): 2D array containing the transformation coefficients. Can be calculated by using:
+            nii = nib.load("nii_path")
+            affine = nii.affine
+        mask (numpy.ndarray, optional): numpy array of booleans with shape of `complex_array` to mask during phase
+                                        unwrapping
+        threshold: Threshold value for automatic mask generation (Use either mask or threshold, not both)
+        is_unwrapping_in_2d (bool, optional): prelude parameter to unwrap slice by slice
+
+    Returns:
+        numpy.ndarray: 3D array with the shape of `complex_array` of the unwrapped phase output from prelude
+    """
+    # Make sure phase and mag are the right shape
+    if wrapped_phase.ndim not in [2, 3]:
+        raise RuntimeError('wrapped_phase must be 2d or 3d')
```

Comment:
convention " vs. ' --> https://github.com/neuropoly/spinalcordtoolbox/wiki/Single-or-double-quotes
```suggestion
        raise RuntimeError("Wrapped_phase must be 2d or 3d")
```